### DISCUSSION TITLE
LS22 Update

### DISCUSSION THREAD

**Discussion by timmi4026** [Mogli12/KeyboardSteer#753] [General]
Moin, es gab einen Patch von Giants, und nun kann ich teilweise nicht mehr vor und zurück fahren und lenken über das G29 wenn ich VCA aktiv habe und es abschalte. Muss es immer wieder an und ausmachen. Kommen keine log Fehler aber die Schaltung undLlenkung sperren ab und zu.

LG Timmi

Nachtrag: Ich beobachte es mal. Kann sein das es nicht an VCA liegt

Also bei mir lag es wohl an Giants, hab das Lenkrad 3x neu in die Keyboardbindings eingetragen und jetzt läuft es. VCA hat bisher auch alles gemacht wie es soll.

**Comment by Mogli12** [comment]
Die Installation des Patches bricht bei mir ständig ab.

**Comment by Sandman92** [comment]
habe seit dem patch wieder das problem mit dem richtungswechsel mit vca das er kein gas mehr annimmt :)

**Comment by Mogli12** [comment]
@Sandman92 Bitte ein paar mehr Details. Welches Fahrzeug? Steht Richtungswechsel auf manuell? Rückfahreinrichtung? Tritt das Problem reproduzierbar nur mit VCA, aber nicht ohne VCA auf?

**Comment by Sandman92** [comment]
Test Fahrzeug Ponsse Buffalo - 900 Serie Fendt bei beiden Rückfahreinrichtung benutzen 
vca Version 9.0.2.5 
Richtungswechel  Auto  in und gegen Fahrtrichtung - kein fehler mit und ohne VCA 
Richtungswechsel Manu - in Fahrtrichtung - kein Fehler  
Richtungswechsel Manu - gegen Fartrichtung -Fehler kein Gas 1kmh

**Comment by Bomberman82** [comment]
Hallo nur zur Info. Dank des neuen Patches sind die Inputbindings total durcheinder gewürfelt. Ich musste alles nochmals neu einstellen. Deswegen kann es sein das diverse Eingaben doppeltgemoppelt sind und das Spiel nicht auseinanderhalten kann, ich würde jedem empfehlen, falls noch nicht gemacht wurde alles auf Standard zu setzen und die Inputbindings nochmals neu einzurichten. 

Und bitte den Mod "Bessere Kabinenansicht" auslassen, weil dieser auch die Inputs ändert. Also ich habe damit z.B. keine Kupplung oder diverse Funktionen nicht mehr. Deswegen habe ich es ausgelassen.

**Comment by crane-denny** [comment]
For everyone who is looking for why VCA is not running anymore after the update.
Like @Bomberman82 already statet. Turn off `Bessere Kabinenansicht`.

**Comment by Mogli12** [comment]
For me it worked to reset the input bindings.

**Comment by crane-denny** [comment]
> For me it worked to reset the input bindings.

Hmm... I didn't tested that. I just removed the mod and everything worked again so I was happy.
